src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -20,6 +20,10 @@ trait ThurloeService extends HttpService {
   val dataAccess: DataAccess
   val thurloePrefix = "thurloe"
 
+  // TODO: Relies on swagger existing in resources/swagger/
+  val swaggerSite = path("swagger-site") { getFromResource("swagger/index.html") } ~ getFromResourceDirectory("swagger")
```

Comment:
Yeah, this whole thing is confusing and isn't helped by it being ill defined currently in the dept. There's a ticket now to get us conforming to the standard and we can address it when we get there.